openvdb_ax/openvdb_ax/compiler/Compiler.h

```diff
@@ -0,0 +1,116 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+/// @file compiler/Compiler.h
+///
+/// @authors Nick Avramoussis, Francisco Gochez, Richard Jones
+///
+/// @brief  The OpenVDB AX Compiler class provides methods to generate
+///   AX executables from a provided AX AST (or directly from a given
+///   string). The class object exists to cache various structures,
+///   primarily LLVM constructs, which benefit from existing across
+///   additional compilation runs.
+///
+
+#ifndef OPENVDB_AX_COMPILER_HAS_BEEN_INCLUDED
+#define OPENVDB_AX_COMPILER_HAS_BEEN_INCLUDED
+
+#include "../ast/AST.h"
+#include "../compiler/CompilerOptions.h"
```

Comment:
Probably left over from an old refactor - but this relative include isn't needed - should simply be "CompilerOptions.h"

Reply:
Fixed